core/src/main/java/org/mobilitydata/gtfsvalidator/annotation/GtfsEnumValues.java

```diff
@@ -0,0 +1,12 @@
+package org.mobilitydata.gtfsvalidator.annotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Target(ElementType.TYPE)
+@Retention(RetentionPolicy.SOURCE)
+public @interface GtfsEnumValues {
```

Comment:
Could you please add documentation explaining how this annotation must be used if it can be used and what's the difference with GtfsEnumValue?

Reply:
It must not be used. Added documentation.